website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
Consolidated in 87fa33c2309626dd99b2ac05ed20708ddb5bc7a4:

website/addons/dataverse/views/crud.py
website/addons/figshare/views/crud.py
website/addons/github/views/crud.py
website/addons/osffiles/views.py
website/addons/wiki/views.py
website/project/views/comment.py
website/project/views/comment.py
website/project/views/contributor.py
website/project/views/contributor.py
website/project/views/node.py
website/project/views/node.py